src/store/foodprint-context.tsx

```diff
@@ -82,9 +82,9 @@ export const FoodprintContext = React.createContext<FoodprintContextObj>({
 
 const FoodprintContextProvider: React.FC = (props) => {
   const [ state, dispatch ] = useReducer(reducer, initialState);
-
   useEffect(() => {
-    const cookieCheck = async () => {
+    if (state.isLoggedIn) {
```

Comment:
You can destructure isLoggedIn from the “state” above so you don’t need to chain ‘. ‘